tests/server/managers/AppAccessorManager.spec.ts

```diff
@@ -127,7 +127,7 @@ export class AppAccessorManagerTestFixture {
         Expect(this.bridges.getServerSettingBridge).toHaveBeenCalled().exactly(1);
         Expect(this.bridges.getEnvironmentalVariableBridge).toHaveBeenCalled().exactly(1);
         Expect(this.bridges.getPersistenceBridge).toHaveBeenCalled().exactly(1);
-        Expect(this.bridges.getRoomBridge).toHaveBeenCalled().exactly(1);
+        Expect(this.bridges.getRoomBridge).toHaveBeenCalled().exactly(2);
```

Comment:
Is it really needed to be called twice? Can't we reuse the first return value from getRoomBridge on both cases?

Reply:
Thanks for the clarification!